src/net_processing.cpp

```diff
@@ -379,7 +379,8 @@ class PeerManagerImpl final : public PeerManager
     /** The height of the best chain */
     std::atomic<int> m_best_height{-1};
 
-    int64_t m_stale_tip_check_time; //!< Next time to check for stale tip
+    /** Next time to check for stale tip */
+    int64_t m_stale_tip_check_time{0};
```

Comment:
754bdaa61fe6473ca7cf9e89146eb8419fdd7c0a

I'm not an expert in Doxygen comments, but our [Developer Notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments) suggest `//!` or `//!<` for variables.

Reply:
Hmm, I wasn't aware of that guidance in the developer notes. Perhaps it should be updated - `/** comment */` is perfectly acceptable to doxygen, and many of the variables/members in the code base use that style.